traffic_ops/traffic_ops_golang/auth/login.go

```diff
@@ -107,7 +108,10 @@ func LoginHandler(db *sqlx.DB, cfg config.Config) http.HandlerFunc {
 
 func CheckLocalUserIsAllowed(form passwordForm, db *sqlx.DB) (bool, error) {
 	var roleName string
-	err := db.Get(&roleName, "SELECT role.name FROM role INNER JOIN tm_user ON tm_user.role = role.id where username=$1", form.Username)
+	dbCtx, dbClose := context.WithTimeout(context.Background(), time.Second*20)
```

Comment:
Can you put this in a config? It'd be super-helpful for debugging to be able to change it dynamically in an installation, without a new binary/RPM.

Reply:
+1, I will update this PR.